src/main/java/frc/robot/subsystems/intakeLimelight/IntakeLimelightConstants.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.subsystems.intakeLimelight;
+
+import frc.robot.Constants;
+
+public class IntakeLimelightConstants {
+  public static final String INTAKE_LIMELIGHT_NAME = "intake";
```

Comment:
I think the name is always `limelight-XYZ` so `limelight-intake`?

Reply:
Good catch